model/Core/Classes/DefinedType.md

```diff
@@ -18,7 +18,7 @@ It provides a structured way to represent defined types, holds information about
 
 ## Properties
 
-- type
+- typeFromSouce
```

Comment:
```suggestion
- typeFromSource
```

model/Core/Properties/typeFromSouce.md also need the same typo fix + file renaming.